src/storage/heap_file.c

```diff
@@ -12715,6 +12715,15 @@ heap_attrinfo_generate_key (THREAD_ENTRY * thread_p, int n_atts, int *att_ids, i
 
   if (func_index_info)
     {
+      if (func_index_info->expr)
+	{
+	  if (heap_attrinfo_read_dbvalues (thread_p, cur_oid, recdes, NULL,
+					   func_index_info->expr->cache_attrinfo) != NO_ERROR)
```

Comment:
the function **heap_attrinfo_read_dbvalues** doesn't use **scan_cache** which is an argument by calling the function.

Reply:
the cur_oid is not used after this routine.

Reply:
Removing an argument from the function heap_attrinfo_read_dbvalues() is not relevant to this issue.

However, it is obvious that it is unnecessary, so I will revise it together.